lib/src/common/model/payment_method_configurations/card_configuration.dart

```diff
@@ -9,6 +9,7 @@ class CardConfiguration {
   final FieldVisibility kcpFieldVisibility;
   final FieldVisibility socialSecurityNumberFieldVisibility;
   final List<String?> supportedCardTypes;
+  final CardCallbacksInterface? cardCallbacks;
```

Comment:
Not sure how Dart's memory model works, but in Android saving callbacks in a field will likely cause a memory leak. What could happen is that this configuration object is not garbage collected before the objects referenced in the callback causing the callback to leak. Or if they do UI related things it could be the callback triggers while the UI is gone and then crashes happen.

In the Android SDK we just pass the callbacks around for this reason. If what I said before applies, would it be possible to also do that?